qr-code-auth/README.md

```diff
@@ -0,0 +1,11 @@
+# QrCodeAuth
+
+```
+{
+  'user_id': string,
+  'device_info':string,
+  'confirmation': boolean,
```

Comment:
looks like we should update this key since it lacks clarity. This key is utilized for confirming the user's selection when they choose "yes" on the popup after scanning the displayed device information on my-site.

Reply:
named to is_authorized as discussed https://discord.com/channels/673083527624916993/735598747949465640/1107838926397382656